content/en/integrations/cloud_foundry.md

```diff
@@ -153,6 +153,14 @@ cf set-env app01 STD_LOG_COLLECTION_PORT 10514
 cf set-env app01 LOGS_CONFIG '[{"type":"tcp","port":"10514","source":"java","service":"app01"}]'
 ```
 
+**Notification in case of misconfigured proxy**
```

Comment:
```suggestion
##### Notification in case of misconfigured proxy
```

I suggest changing all bolded titles in this section so they are clickable links